flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/table/TableSinkITCase.scala

```diff
@@ -177,10 +195,16 @@ class TableSinkITCase extends AbstractTestBase {
       .assignAscendingTimestamps(_._1.toLong)
       .toTable(tEnv, 'id, 'num, 'text)
 
+    tEnv.registerTableSink(
+      "retractSink",
+      new TestRetractSink().configure(
+        Array[String]("len", "icnt", "nsum"),
+        Array[TypeInformation[_]](Types.INT, Types.LONG, Types.LONG)))
+
     t.select('id, 'num, 'text.charLength() as 'len)
       .groupBy('len)
-      .select('len, 'id.count, 'num.sum)
-      .writeToSink(new TestRetractSink)
+      .select('len, 'id.count as 'icnt, 'num.sum as 'nsum)
```

Comment:
In order to show that the column name of the source table can be inconsistent with the column name of the sink table, I recommend removing the alias. What do you think？

Reply:
I'd keep it here for clarity.
There are other tests that check that field names do not need to match like `TableSinkITCase.testInsertIntoRegisteredTableSink()`.

Reply:
Sounds good to me.